### ISSUE TITLE
Fails to checkin 

### ISSUE THREAD

**Issue by prajwalbasnet4400** [pyro2927/SouthwestCheckin#75]
Returns a 429 response with this body
`{
  "code": 429999999,
  "message": "Error.",
  "messageKey": "ERROR",
  "httpStatusCode": "BAD_REQUEST",
  "requestId": "",
  "infoList": []
}`
and response headers:
`{'Content-Type': 'text/plain; charset=UTF-8', 'Content-Length': '163', 'X-Ion-Hop': '1', 'Expires': 'Wed, 13 Apr 2022 09:35:39 GMT', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Pragma': 'no-cache', 'Date': 'Wed, 13 Apr 2022 09:35:39 GMT', 'Connection': 'close', 'Set-Cookie': 'sRpK8nqm_sc=A21LRyKAAQAAT0AFOnsQkl1ODo8qf8IwHSsdb-GiWlvMxxYtZWfIer_tienRAS1X1nTURuIXFooAAAAAAAAAAA|1|0|bfbddfacd00dffc14ffd69663365d3a90b463ae0; Path=/; Max-Age=1577847600; Domain=southwest.com; Secure', 'X-Frame-Options': 'SAMEORIGIN', 'Strict-Transport-Security': 'max-age=31536000', 'terms-of-service': "Unauthorized access, display, or use of Southwest's Company Information, including fare data, is prohibited by the Terms & Conditions on Southwest.com and Swabiz.com."}`

**Comment by sabujp** [comment]
Confirmed this doesn't work:

```
Flight information found, departing XXX, XX, at XXX XX XX:XXAM
Too early to check in.  Waiting 0 hours, 9 minutes, 20.37888399999997 seconds
Sorry! This reservation is not eligible for check in.
Sorry! This reservation is not eligible for check in.
This flight is currently not ready for check-in. Please try again momentarily.
This flight is currently not ready for check-in. Please try again momentarily.
Attempting check-in...
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
Error.
```

**Comment by rodwhis** [comment]
Yes, I got the same results

**Comment by Evilgeniusnerd** [comment]
I got this same error. I know when I checked in for my flight manually there was some pop-up to confirm you aren't sick with Coivd-19 etc... So I presume that pop-up is causing the code to crash since the current program doesn't know to go in and click the prompt? I am no programmer but that's my two cents.

**Comment by mkitchingh** [comment]
I don't believe that pop-up is the cause of your error. That particular pop-up has been there for over a year.

**Comment by h4xdaplanet** [comment]
If you are not scraping headers via what has been discussed - then that is
why it is not working.

**Comment by akash329d** [comment]
@h4xdaplanet what headers are you referring to? Haven’t seen any discussion on that.

**Comment by h4xdaplanet** [comment]
Take a look at #70 and this comment specifically

https://github.com/pyro2927/SouthwestCheckin/issues/70#issuecomment-991957843
